### DISCUSSION TITLE
Shouldn't .m4a import be automatic?

### DISCUSSION THREAD

**Discussion by Krythz43** [audacity/audacity#6789] [Ideas & Feature Requests]
### Your idea

Since Apple airdrops come with .m4a file format while recording audio memos, should't Audacity be capable of automatically importing it? This spoiled my user experience and probably will for a lot of people after me too.

<img width="622" alt="image" src="https://github.com/audacity/audacity/assets/33388337/6b70696c-46f6-4ca6-812c-6bda26609475">

### Problem to be solved

Auto import of .m4a

**Comment by petersampsonaudacity** [comment]
There are licensing, and thereby cost, issues that prevent FFmpeg being bundled with Audacity.  

The same used to be the same for LAME (needed for MP3 import/export), but the Fraunhofer patents have expired as of April 2017, enabling LAME to be bundled as part of Audacity with no cost implications and this was the case for Audacity 2.3.2 onwards.

See this page on the Muse Support site:
https://support.audacityteam.org/basics/installing-ffmpeg
